plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduAuthenticationType.java

```diff
@@ -0,0 +1,18 @@
+package io.trino.plugin.kudu;
+
+import static java.util.Locale.ENGLISH;
+
+public enum KuduAuthenticationType
+{
+    NONE,
+    KERBEROS;
+
+    @SuppressWarnings("unused") // used by airlift
+    public static KuduAuthenticationType fromString(String value)
```

Comment:
Do we need this snippet ? Won't airlift maps String to the enum ?

Reply:
Good question, I lifted/shifted this from another connector. I'll test and verify.

Reply:
Not needed